src/javascript/_common/third_party_links.js

```diff
@@ -81,6 +81,7 @@ const ThirdPartyLinks = (() => {
             && !new RegExp(`^.*\\.${getCurrentBinaryDomain() || 'binary\\.com'}$`).test(destination.host) // destination host is not binary subdomain
             && !new RegExp('^.*\\.binary\\.bot$').test(destination.host) // destination host is not binary subdomain
             && !/www.(betonmarkets|xodds).com/.test(destination.host) // destination host is not binary old domain
+            && !/www.deriv.(app|com)/.test(destination.host) // destination host is not deriv
```

Comment:
as far as i know, the dns has changed the hostname to be `deriv.xx` removing `www`